### DISCUSSION TITLE
Clear the Arduino IDE Tools Menu Cache?

### DISCUSSION THREAD

**Discussion by Xylopyrographer** [espressif/arduino-esp32#10056] [Question - Community Help]
I'm working on adding support for a board.

In the `variants` folder for the board are a couple of custom partition files.

In the section of the `boards.txt` modified to add this new board, I've added the description for these custom partition files.

On launching the Arduino IDE, all is good - the new board can be selected and the "Tools" menu shows what it should for the board specific items including the custom partitions.

However, after making changes to the section for this board in `boards.txt`, those changes are not reflected when relaunching the IDE. Specifically, the "Tool" menu is the same as it was for this board when it was first added to `boards.txt`.

_Is there a cache somewhere that needs to be deleted?_

FWIW, I am clearing the `/tmp/arduino*` folders after making changes and before relaunching the IDE. No effect on the "Tools" menu.

There is also `~/.arduino15/tmp` folder but that seems to auto-clear on closing the IDE.

System: Debian/GNU Linux 12
Arduino IDE v2.32
esp32-arduino core v3.0.3

Let me know if any other info would be useful.

Many thanks.

**Comment by me-no-dev** [comment]
IDE2 has additional cache for that, though I do not recall where it was. Maybe google can help or @pillo79 ?

**Comment by Xylopyrographer** [comment]
Found the secret cache...

Known issue with Arduino IDE 2. See:  https://github.com/arduino/arduino-ide/issues/1030

Workaround is: https://github.com/arduino/arduino-ide/issues/1030#issuecomment-1152005617
